docs/release-notes/2025-03-24-discontinuation-of-the-pricing-v1-module-compatiblity-routes.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Discontinuation of the Pricing V1 module compatibility routes"
+slug: "2025-03-24-discontinuation-of-the-pricing-v1-module-compatiblity-routes"
+hidden: false
+type: "deprecated"
+createdAt: "2025-03-24T00:00:00.219Z"
+updatedAt: ""
+excerpt: "After July 1, 2025, any store using the compatibility routes of the Pricing V1 module will get the following error message: This resource is not available for this account, with status HTTP 410 Gone"
+---
+
+After July 1, 2025, any store using the compatibility routes of the Pricing V1 module will get the following error message:
+
+>❗ This resource is not available for this account, with status HTTP 410 Gone
+
+## What has changed?
+We've disabled the compatibility routes of the Pricing V1 module to simplify maintenance and focus on evolving the platform. This change requires all clients to adopt the routes of the Pricing V2 module, which ensures access to updates and new features.
+
+This action is part of the deprecation process of the previous version of the module, announced in [Pricing V1 module will be discontinued](https://help.vtex.com/en/announcements/deprecacao-pricing-v1--46YxKNOCLH2Ykw6a9uyxXB?&utm_source=autocomplete).
```

Comment:
Remover `?&utm_source=autocomplete` do link.

Reply:
A slug correta para a versão EN é: https://help.vtex.com/en/announcements/pricing-v1-module-will-be-discontinued-update-your-store-by-september--46YxKNOCLH2Ykw6a9uyxXB